--- env.rhino.js.org	2009-02-27 11:53:23.000000000 +0100
+++ env.rhino.js	2009-02-27 11:51:35.000000000 +0100
@@ -16,7 +16,7 @@
     
     $env.log = function(){};
     //uncomment this if you want to get some internal log statementes
-    $env.log = print;
+    //$env.log = print;
     $env.log("Initializing Rhino Platform Env");
     
     $env.error = function(msg, e){
@@ -2038,15 +2038,15 @@
 });    
 
 $log("Defining Element");
-/*
-* Element - DOM Level 2
-*/
-$w.__defineGetter__("Element", function(){
-  return function(){
-    throw new Error("Object cannot be created in this context");
-  };
-});
-
+// /*
+// * Element - DOM Level 2
+// */
+// $w.__defineGetter__("Element", function(){
+//   return function(){
+//     throw new Error("Object cannot be created in this context");
+//   };
+// });
+var Element = { Methods: {} };
 	
 /**
  * @class  DOMElement - By far the vast majority of objects (apart from text) that authors encounter
@@ -5799,11 +5799,12 @@
 /* 
 * HTMLAnchorElement - DOM Level 2
 */
-$w.__defineGetter__("Form", function(){
-  return function(){
-    throw new Error("Object cannot be created in this context");
-  };
-});
+// $w.__defineGetter__("Form", function(){
+//   return function(){
+//     throw new Error("Object cannot be created in this context");
+//   };
+// });
+var Form = {};
 $w.__defineGetter__("HTMLFormElement", function(){
   return function(){
     throw new Error("Object cannot be created in this context");
