--- env.rhino.0.9.5.js.orig	2009-05-25 11:30:23.000000000 +0200
+++ env.rhino.js	2009-05-25 11:31:25.000000000 +0200
@@ -165,7 +165,7 @@
 (function($env){
     
     $env.log = function(msg, level){
-         print(' '+ (level?level:'LOG') + ':\t['+ new Date()+"] {ENVJS} "+msg);
+//         print(' '+ (level?level:'LOG') + ':\t['+ new Date()+"] {ENVJS} "+msg);
     };
     
     $env.lineSource = function(e){
@@ -2193,11 +2193,11 @@
 /*
 * Element - DOM Level 2
 */
-$w.__defineGetter__("Element", function(){
-  return function(){
-    throw new Error("Object cannot be created in this context");
-  };
-});
+// $w.__defineGetter__("Element", function(){
+//   return function(){
+//     throw new Error("Object cannot be created in this context");
+//   };
+// });
 
 	
 /**
@@ -5988,11 +5988,11 @@
 /* 
 * HTMLAnchorElement - DOM Level 2
 */
-$w.__defineGetter__("Form", function(){
-  return function(){
-    throw new Error("Object cannot be created in this context");
-  };
-});
+// $w.__defineGetter__("Form", function(){
+//   return function(){
+//     throw new Error("Object cannot be created in this context");
+//   };
+// });
 $w.__defineGetter__("HTMLFormElement", function(){
   return function(){
     throw new Error("Object cannot be created in this context");
