--- env.rhino.0.9.5.js.orig	2009-05-25 11:30:23.000000000 +0200
+++ env.rhino.js	2009-05-25 11:31:25.000000000 +0200
@@ -165,7 +165,7 @@
 (function($env){
     
     $env.log = function(msg, level){
-         print(' '+ (level?level:'LOG') + ':\t['+ new Date()+"] {ENVJS} "+msg);
+//         print(' '+ (level?level:'LOG') + ':\t['+ new Date()+"] {ENVJS} "+msg);
     };
     
     $env.lineSource = function(e){
